{{ cookiecutter.dir_name }}/build.gradle

```diff
@@ -60,6 +60,10 @@ project.afterEvaluate {
 }
 
 dependencies {
+    repositories {
+        jcenter()
+    }
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:25.1.1'
+    compile 'com.android.volley:volley:1.1.0'
```

Comment:
As with the INTERNET permission - is this something we want to include in *every* app? Sure, a lot of apps will have use for a JSON request mechanism - but will *every* app need this code? If not, then this needs to be factored into an option somewhere.